FC_patch_sequence_for_dummy_verify.sh

```diff
@@ -121,7 +125,8 @@ echo "               Patching flight parameters hardcoded values"
 echo "               Adding U-Blox custom configuration"
 echo "################################################################################"
 
-"$PATH_TO_TOOLS"/FC_Patcher/patch_"$AC_PREFIX"_"$MODULE".py $TMP_FILENAME1 $VERSION
+# call it using python version 2
+python2 "$PATH_TO_TOOLS"/DJI_FC_Patcher/patch_"$AC_PREFIX"_"$MODULE".py $TMP_FILENAME1 $VERSION
```

Comment:
some tools are using python3 best way is to run then within venv, then `python` env is overwritten by `python3` bin. This way we'll call `python` in version 2 (which is required by patch_x.py scripts)